packer/ansible/roles/tomcat/files/server.xml

```diff
@@ -1,168 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I typically like keep the stock file and making the necessary changes. I think it makes for easier diffs with the stock file on a default install and the links to the docs are nice too.

But I'm guessing you removed this to make the grepping easier?

Reply:
That's exactly right. I'll add the stock file as well with a `.original` prefix, just in case